src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -488,9 +490,11 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_get(void *origin_addr,
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_MPI_GET);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_GET);
 
-    if (!MPIDI_OFI_ENABLE_RMA || !(winattr & MPIDI_WINATTR_NM_REACHABLE)) {
-        mpi_errno = MPIDIG_mpi_get(origin_addr, origin_count, origin_datatype, target_rank,
-                                   target_disp, target_count, target_datatype, win);
+    if (!MPIDI_OFI_ENABLE_RMA || !(winattr & MPIDI_WINATTR_NM_REACHABLE) ||
+        MPIR_GPU_query_pointer_is_dev(origin_addr)) {
+        mpi_errno =
+            MPIDIG_mpi_get(origin_addr, origin_count, origin_datatype, target_rank, target_disp,
+                           target_count, target_datatype, win);
         goto fn_exit;
```

Comment:
A minor comment: we might manually reformat line 496 to avoid code cleanup change. See below:
```
        mpi_errno = MPIDIG_mpi_get(origin_addr, origin_count, origin_datatype, target_rank,
                                                          target_disp, target_count, target_datatype, win);
```

Same for the following lines:
- line 331 `mpi_errno = MPIDIG_mpi_put...`
- line 530 `mpi_errno = MPIDIG_mpi_rput...`
- line 1161 `mpi_errno = MPIDIG_mpi_rget...`

Reply:
Fixed.

Reply:
It looks like the `ch4/ofi: Fallback to generic RMA with GPU buffers` commit is not updated. I still see the format-only changes.

Reply:
Oops. Should be fixed now.